packages/library-base/src/commonMain/kotlin/io/realm/kotlin/schema/RealmClassKind.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.kotlin.schema
+
+/**
+ * Enum describing what kind of Realm object it is.
+ */
+public enum class RealmClassKind {
+    /**
+     * Standard Realm objects are the default kind of object in Realm, and they extend the
+     * [io.realm.kotlin.types.RealmObject] interface.
+     */
+    STANDARD,
```

Comment:
Yeah, I wasn't 100% sure whether to use NORMAL or STANDARD here. I don't think other SDKs expose something like this, so it is up to us. I think I opted for STANDARD since that was the name Core used and NORMAL didn't seem much better. Not sure how you feel?

Reply:
Ok, maybe core is not even consistent ... I was just suggesting to align it with https://github.com/realm/realm-core/blob/master/src/realm.h#L302, but if they use _standard_ objects elsewhere then let's just leave it 😅